forex-service/src/main/java/lk/ac/ucsc/forexservice/ForexController.java

```diff
@@ -0,0 +1,28 @@
+package lk.ac.ucsc.forexservice;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ForexController {
+    @Autowired
+    private Environment environment;
+
+    @Autowired
+    private ExchangeValueRepository repository;
+
+    @GetMapping("/currency-exchange/from/{from}/to/{to}")
+    public ExchangeValue getExchangeValue(@PathVariable("from") String from, @PathVariable("to") String to) {
+//        return repository.findByFromAndTo(from, to);
+        ExchangeValue exchangeValue =
+                repository.findByFromAndTo(from, to);
+
+        exchangeValue.setPort(
+                Integer.parseInt(environment.getProperty("local.server.port")));
+
+        return exchangeValue;
```

Comment:
@Binurim u cn directly return the previous statement

Reply:
How? it gives an error neh.